src/main/java/frc/robot/commands/RotateWristCommand.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ArmSubsystem;
+
+public class RotateWristCommand extends CommandBase {
+    private final ArmSubsystem arms;
+    private final double power;
+
+    public RotateWristCommand(ArmSubsystem arms, double power) {
```

Comment:
What's the rationale behind having one single `ArmSubsystem` for both the shoulder and wrist, but two separate commands `RotateArmCommand` and `RotateWristCommand`? We should decide if the arm and wrist are separate entities, or two parts of the same entity, then keep that decision consistent throughout the code.